podcastScreen/podcast_screen/lib/main.dart

```diff
@@ -54,107 +54,96 @@ class _PodcastPageState extends State<PodcastPage> {
           ),
           SliverToBoxAdapter(
             child: Container(
+              padding: const EdgeInsets.all(16.0),
               child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
                 children: <Widget>[
                   _buildPodcastCard(),
-                  SizedBox(
```

Comment:
SizedBox is no longer required as there is already proper spacing between the podcastCards.